ci/rpmostree.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# Build and run rpm-ostree's unit tests using the just-built ostree for this PR.
+
+set -xeuo pipefail
+
+# Frozen to a tag for now to help predictability; it's
+# also useful to test building *older* versions since
+# that must work.
+RPMOSTREE_TAG=v2017.11
+
+dn=$(dirname $0)
+. ${dn}/libbuild.sh
+
+codedir=$(pwd)
+
+pkg_upgrade
+pkg_install_builddeps ostree
+pkg_install_builddeps rpm-ostree
+pkg_install rpm-ostree && rpm -e rpm-ostree
+
+# Duplicate of deps from build.sh in rpm-ostree for tests
+pkg_install ostree{,-devel,-grub2} createrepo_c /usr/bin/jq PyYAML \
+            libubsan libasan libtsan elfutils fuse sudo python-gobject-base \
+            selinux-policy-devel selinux-policy-targeted
+
+# build+install ostree
+cd ${codedir}
+build
+make install
+
+tmpd=$(mktemp -d)
+cd ${tmpd}
+git clone --recursive --depth=1 -b ${RPMOSTREE_TAG} https://github.com/projectatomic/rpm-ostree
+cd rpm-ostree
+build
+# We want to capture automake results
+cleanup() {
+    mv test-suite.log ${codedir} || true
+}
+trap cleanup EXIT
+make -j 8 check
+# Just a subset ⊂ for now to avoid CI overload
+#make vmcheck TESTS=layering-basic
```

Comment:
Let's at least run `test-basic` and/or `test-layering-basic`? Otherwise, there's no point in provisioning a host too, right?

Reply:
Yeah the idea was to do the unit tests first.  I'll can take a stab at doing those two in this PR, or do you think we should do a separate PR?

Reply:
Doing it in a separate PR is cool with me! Though maybe let's drop the host provisioning in this PR then?

Reply:
Eh let's do it in this one.  Working on it now.